distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numeric/Math_Context.enso

```diff
@@ -0,0 +1,26 @@
+import project.Data.Numeric.Rounding_Mode.Rounding_Mode
+from project.Data.Numbers import Integer
+
+polyglot java import java.math.MathContext
+
+## A wrapper around the Java
+   [`MathContext`](https://docs.oracle.com/javase/8/docs/api/java/math/MathContext.html)
+   class.
+
+   `Math_Context` is used to specify finite or infinite precision requirements
+   to operations that construct `Decimal` values, or that compute them from
+   other values. It can also specify the `Rounding_Mode` to use when rounding is
+   used to meet precision requirements. See `Decimal` for more details.
+type Math_Context
+    ## PRIVATE
+    Value (math_context:MathContext)
+
+    ## Construct a `Math_Context` value.
+
+       Arguments:
+       - precision: The non-negative int precision setting. A value of `0`
+         specifies exact operations.
+       - rounding_mode: The rounding mode to use.
+    new : Integer -> Rounding_Mode -> Math_Context
+    new (precision:Integer) (rounding_mode:Rounding_Mode)=Rounding_Mode.half_up =
```

Comment:
I'm usually advocating for the `(_ : _ = _)` signature style

```suggestion
    new (precision:Integer) (rounding_mode : Rounding_Mode = Rounding_Mode.half_up) =
```

Reply:
Done